gems/sorbet-runtime/lib/types/_types.rb

```diff
@@ -88,8 +88,12 @@ def self.class_of(klass)
   #
   # The name of the type alias is not preserved; Error messages will
   # be printed with reference to the underlying type.
-  def self.type_alias(type)
-    T::Utils.coerce(type)
+  def self.type_alias(type=nil, &blk) # TODO TODO TODO
```

Comment:
Nice! We'll need to describe the change to open-source folks.
Are we also writing a rubocop for automatic conversion? Can we opensource it?

Reply:
I have a rubocop autocorrector. Have we shipped those in the sorbet repo itself in the past?

Reply:
The last similarly big change of syntax happened before we open-sourced, so, at the time, we just dropped it into slack.

Now that I think of it, it might be better to do it as a Sorbet autocorrect as they we'll have an easy distribution mechanism.

Reply:
I'll look into that

Reply:
We open-sourced a set of [Rubocops for Sorbet](https://github.com/Shopify/rubocop-sorbet) if you want to add yours waiting for Sorbet's autocorrect?

Reply:
I was able to turn this into a Sorbet autocorrector.